exercise1.py

```diff
@@ -0,0 +1,50 @@
+# task 1
+import datetime
+import this  # appeared to be some pseudo philosophy by some dinosaur developer
```

Comment:
It's real philosophy)
if you think deaper what is written there - it will really help you in future coding, when you are doubting between two implementations for example. Or even in everyday coding.